x-pack/plugins/security_solution/public/detection_engine/rule_management_ui/components/maintenance_window_callout/maintenance_window_callout.test.tsx

```diff
@@ -0,0 +1,223 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { render, waitFor, cleanup } from '@testing-library/react';
+import { QueryClient, QueryClientProvider } from '@tanstack/react-query';
+import { useAppToasts } from '../../../../common/hooks/use_app_toasts';
+import { useAppToastsMock } from '../../../../common/hooks/use_app_toasts.mock';
+import { MaintenanceWindowCallout } from './maintenance_window_callout';
+import { TestProviders } from '../../../../common/mock';
+import { fetchActiveMaintenanceWindows } from './api';
+
+jest.mock('../../../../common/hooks/use_app_toasts');
+
+jest.mock('./api', () => ({
+  fetchActiveMaintenanceWindows: jest.fn(() => Promise.resolve([])),
+}));
+
+const RUNNING_MAINTENANCE_WINDOW_1 = {
```

Comment:
It's hard to distinguish `RUNNING_MAINTENANCE_WINDOW_1` and `RUNNING_MAINTENANCE_WINDOW_2` by name. It can be renamed to something like `ONE_RUNNING_MAINTENANCE_WINDOW_DATA` and `MULTIPLE_RUNNING_MAINTENANCE_WINDOW_DATA`.

Reply:
My idea here was to test that the same callout is shown no matter how many MWs are currently running. So I mocked two running MWs -`RUNNING_MAINTENANCE_WINDOW_1` and `RUNNING_MAINTENANCE_WINDOW_2`. Each of these is a single MW with a "running" status, just a slight difference in timeframe.  

Then I ran one test with:
```typescript
(fetchActiveMaintenanceWindows as jest.Mock).mockResolvedValue([
  RUNNING_MAINTENANCE_WINDOW_1
]);
```
and another one with:
```typescript
(fetchActiveMaintenanceWindows as jest.Mock).mockResolvedValue([
  RUNNING_MAINTENANCE_WINDOW_1,
  RUNNING_MAINTENANCE_WINDOW_2,
]);
```

Does it make sense? wdyt?

Reply:
Sounds good